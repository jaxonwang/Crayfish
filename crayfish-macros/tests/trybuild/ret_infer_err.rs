use crayfish_macros::*;

#[activity]
#[allow(unused_variables)]
fn bar_impl(a: i32) -> impl Fut<Output=usize>{
    async {
        123
    }
}

pub fn main() {}
