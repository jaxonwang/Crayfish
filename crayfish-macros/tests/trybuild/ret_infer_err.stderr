error: can not infer return type. Please set a attribute: #[activity(ret = "Type")]
 --> $DIR/ret_infer_err.rs:5:24
  |
5 | fn bar_impl(a: i32) -> impl Fut<Output=usize>{
  |                        ^^^^^^^^^^^^^^^^^^^^^^
